### PR TITLE
Add remote SAML logout request documentation

### PR DESCRIPTION
**Pull request by orenyk**

### CODE REVIEW

File: _pages/saml.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ sidenav:
         href: "#logout-request"
       - text: Logout response
         href: "#logout-response"
+      - text: Remote (back-channel) logout
```
Comment (zachmargolis) [inline _pages/saml.md:29]: style: I don't think backchannel is hyphenated? when I search online, I see "back channel" as a noun, but as a verb it's "backchanneled" no space, ex: https://en.wikipedia.org/wiki/Backchannel_(linguistics)

```suggestion
      - text: Remote (backchannel) logout
```

File: _pages/saml.md [hunk 2]
```diff
@@ -499,6 +501,77 @@ An example decoded logout response, with indentation added for readability.
 </div>
 </div>
 
+### Remote Logout request
+
+Login.gov also offers a remote / back-channel logout endpoint if your
```
Comment (zachmargolis) [inline _pages/saml.md:506]:
```suggestion
Login.gov also offers a remote / backchannel logout endpoint if your
```

same style re: backchannel

File: _pages/saml.md [hunk 3]
```diff
@@ -499,6 +501,77 @@ An example decoded logout response, with indentation added for readability.
...
+application needs to log users out without redirecting them back to Login.gov.
+This is still **not** true Single Logout (SLO), it will only terminate a given
+user's session with Login.gov.
+
+For remote logout, you must include a `SessionIndex` element in the SAML request
+that contains the user's Login.gov UUID. This will allow Login.gov to identify
+the user that needs to be logged out and terminate their session.
+
+To log a user out using a back-channel request, direct them to the logout URL
```
Comment (zachmargolis) [inline _pages/saml.md:515]:
```suggestion
To log a user out using a backchannel request, direct them to the logout URL
```

File: _pages/saml.md [hunk 4]
```diff
@@ -499,6 +501,77 @@ An example decoded logout response, with indentation added for readability.
 </div>
 </div>
 
+### Remote Logout request
+
+Login.gov also offers a remote / back-channel logout endpoint if your
+application needs to log users out without redirecting them back to Login.gov.
+This is still **not** true Single Logout (SLO), it will only terminate a given
+user's session with Login.gov.
```
Comment (zachmargolis) [inline _pages/saml.md:509]: maybe add "you still need to manage the session for your own application" or something?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _pages/saml.md (hunk 1 above)
Comment (orenyk) [reply _pages/saml.md:29]: Thanks! Apparently I've been doing it wrong the whole time 😛

File: _pages/saml.md (hunk 4 above)
Comment (orenyk) [reply _pages/saml.md]: Great point, I'll add it in.

File: _pages/saml.md (hunk 1 above)
Comment (orenyk) [reply _pages/saml.md]: After chatting on Slack we're going to go with a space instead of all one word.

**Comment by orenyk** [comment]
Will merge tomorrow after deployment

**Pull request merged**